src/EditorFeatures/Test2/FindReferences/FindReferencesTests.AccessorSymbols.vb

```diff
@@ -0,0 +1,1685 @@
+        Public Async Function TestCSharpAccessor_Prop_ObjectInitializer1() As Task
+            Dim input =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document>
+class C
+{
+    public virtual int {|Definition:$$Prop|} { get; set; }
+}
+
+class Usages
+{
+    void M()
+    {
+        new C
+        {
+            [|Prop|] = 1
+        };
+    }
+}
+        </Document>
+    </Project>
+</Workspace>
+            Await TestStreamingFeature(input)
+        End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function TestCSharpAccessor_Get_Indexer1() As Task
+            Dim input =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document>
+interface IC
+{
+    int this[int i] { {|Definition:$$get|} => 0; set { } }
+}
+
+class C : IC
+{
+    public virtual int this[int i] { {|Definition:get|} => 0; set { } }
+}
+
+class D : C
+{
+    public override int this[int i] { {|Definition:get|} => base[||][i]; set { base[i] = value; } }
+}
+
+class Usages
+{
+    void M()
+    {
+        IC ic;
+        var v1 = ic[||][0]
+        ic[0] = 1
+
+        C c;
+        var v1 = c[||][0]
+        c[0] = 1
+
+        D d;
+        var v1 = d[||][0]
+        d[0] = 1
+    }
+}
+        </Document>
+    </Project>
+</Workspace>
+            Await TestStreamingFeature(input)
+        End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function TestCSharpAccessor_Get_Indexer2() As Task
+            Dim input =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document>
+interface IC
+{
+    int this[int i] { {|Definition:get|} => 0; set { } }
+}
+
+class C : IC
+{
+    public virtual int this[int i] { {|Definition:$$get|} => 0; set { } }
+}
+
+class D : C
+{
+    public override int this[int i] { {|Definition:get|} => base[||][i]; set { base[i] = value; } }
+}
+
+class Usages
+{
+    void M()
+    {
+        IC ic;
+        var v1 = ic[||][0]
+        ic[0] = 1
+
+        C c;
+        var v1 = c[||][0]
+        c[0] = 1
+
+        D d;
+        var v1 = d[||][0]
+        d[0] = 1
+    }
+}
+        </Document>
+    </Project>
+</Workspace>
+            Await TestStreamingFeature(input)
+        End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function TestCSharpAccessor_Get_Indexer3() As Task
+            Dim input =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document>
+interface IC
+{
+    int this[int i] { {|Definition:get|} => 0; set { } }
+}
+
+class C : IC
+{
+    public virtual int this[int i] { {|Definition:get|} => 0; set { } }
+}
+
+class D : C
+{
+    public override int this[int i] { {|Definition:$$get|} => base[||][i]; set { base[i] = value; } }
+}
+
+class Usages
+{
+    void M()
+    {
+        IC ic;
+        var v1 = ic[||][0]
```

Comment:
>ic[||][0] [](start = 17, length = 9)

You're missing semi-colons (here and a bunch of places).
Also, I'm curious how this scenario (highlight with zero width: `[||]`) displays in the IDE.